src/main/java/easytests/api/v1/controllers/TopicsController.java

```diff
@@ -78,7 +78,28 @@ public Topic show(@PathVariable Integer topicId) throws NotFoundException, Forbi
         }
         return this.topicsMapper.map(topicModel, Topic.class);
     }
-    /**
-     * delete(topicId)
-     */
+
+    @DeleteMapping("/{topicId}")
+    public void delete(@PathVariable Integer topicId) throws NotFoundException, ForbiddenException {
+        final TopicModelInterface topicModel = this.getTopicModel(topicId);
+
+        if (!this.acl.hasAccess(topicModel)) {
+            throw new ForbiddenException();
+        }
+
+        this.topicsService.delete(topicModel);
```

Comment:
that would not work, you should provide correct options for delete object

Reply:
done